src/applications/vaos/tests/new-appointment/components/ConfirmationPage/ConfirmationRequestInfo.unit.spec.jsx

```diff
@@ -189,30 +197,39 @@ describe('VAOS <ConfirmationRequestInfo>', () => {
     };
     const pageTitle = 'Your appointment request has been submitted';
 
-    const tree = mount(
+    const screen = renderWithStoreAndRouter(
       <ConfirmationRequestInfo
         data={data}
         facilityDetails={facilityDetails}
         pageTitle={pageTitle}
       />,
+      {
+        initialState,
+        reducers,
+      },
     );
 
-    tree
-      .find('AdditionalInfo')
-      .find('button')
-      .simulate('click');
-    tree.setProps();
+    // Simulate user clicking the "Show more" button
+    const button = screen.getByRole('button', { name: 'Show more' });
+    userEvent.click(button);
```

Comment:
You can do `screen.click` instead, so you don't have to import `userEvent`

Reply:
Just following suggested guidelines here [](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library)

Reply:
Ah, interesting, I read that article before we started using RTL, I need to read through it again now that we have some experience. No need to change this, then.